lib/temporal/client/serializer/continue_as_new.rb

```diff
@@ -1,6 +1,4 @@
-require 'temporal/client/serializer/base'
-require 'temporal/client/serializer/payload'
```

Comment:
This file is still using `Payload` serializer, which is no longer there. Needs to be replaced and probably needs a spec to confirm